grails-app/migrations/changelog.xml

```diff
@@ -104,4 +104,55 @@
         </createIndex>
     </changeSet>
 
+    <changeSet id="10-prerequisite-index-for-summary-aggregation" author="Qifeng Bai">
```

Comment:
remove this changeset, see ala-install where the identical primary key is created